modules/components/tag-input.ts

```diff
@@ -674,12 +681,19 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
         });
 
         listen.call(this, constants.KEYDOWN, ($event) => {
-            if (this.separatorKeys.indexOf($event.keyCode) >= 0) {
+            if (this.separatorKeys.indexOf($event.key) >= 0) {
```

Comment:
What if this was:

```
const hasKeyCode = this.separatorKeyCodes.indexOf($event.keyCode) >= 0;
const hasKey = this.separatorKeys.indexOf($event.key) >= 0;
if (hasKeyCode >= 0 || hasKey) {
///
} 

so that we can remove the event below?